src/js/components/Menu.js

```diff
@@ -179,6 +179,7 @@ class MenuDrop extends Component {
     }
 
     let classes = classnames(
+      `${this.props.className}__drop`,
```

Comment:
We need to safe guard className since it will not always be present

```
  {
          [`${this.props.className}__drop`]: this.props.className,
```